app.json

```diff
@@ -0,0 +1,279 @@
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com/login-callback"
+    },
+
+    "AUTH0_LOGOUT_CALLBACK": {
+      "description": "auth0 logout callback url",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com/logout-callback"
+    },
+
+    "AUTH0_DOMAIN": {
+      "description": "auth0 domain attached to auth0 account",
+      "required": true,
+      "value": "<changeme>.auth0.com"
+    },
+
+    "AUTH0_CLIENT_ID": {
+      "description": "auth0 client id attached to auth0 account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "AUTH0_CLIENT_SECRET": {
+      "description": "auth0 client secret attached to auth0 account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "BASE_URL": {
+      "description": "based on your herokuapp name or domain name",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com"
+    },
+
+    "GRAPHQL_URL": {
+      "description": "based on your herokuapp name or domain name",
+      "required": false,
+      "value": "https://<changeme>.herokuapp.com/graphql"
+    },
+
+    "SESSION_SECRET": {
+      "description": "session secret",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "DEFAULT_SERVICE": {
+      "description": "specifies using twilio api ",
+      "required": true,
+      "value": "twilio"
+    },
+
+    "TWILIO_API_KEY": {
+      "description": "for twilio integration and connected to twilio account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "TWILIO_APPLICATION_SID": {
+      "description": "for twilio integration and connected to twilio account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "TWILIO_AUTH_TOKEN": {
+      "description": "for twilio integration and connected to twilio account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "TWILIO_MESSAGE_SERVICE_SID": {
+      "description": "for twilio integration and connected to twilio account",
+      "required": true,
+      "value": "<changeme>"
+    },
+
+    "TWILIO_STATUS_CALLBACK_URL": {
+      "description": "for twilio integration and connected to twilio account",
+      "required": true,
+      "value": "https://<changeme>.herokuapp.com/twilio-message-report"
+    },
+
+    "PHONE_NUMBER_COUNTRY": {
+      "description": "for international integration (when not us)",
+      "required": true,
+      "value": "US"
+    },
+
+    "ROLLBAR_CLIENT_TOKEN": {
+      "description": "rollbar api integration",
+      "required": false,
+      "value": ""
+    },
+
+    "ROLLBAR_ACCESS_TOKEN": {
+      "description": "rollbar api integration",
+      "required": false,
+      "value": ""
+    },
+
+    "ROLLBAR_ENDPOINT": {
+      "description": "rollbar api integration",
+      "required": false,
+      "value": "https://api.rollbar.com/api/1/item/"
+    },
+
+    "EMAIL_HOST": {
```

Comment:
Probably something for later, but might be nice to leverage this:
https://devcenter.heroku.com/articles/smtp
e.g. sendgrid:starter seems to be free.  If that just changes WHAT creds you need to put in, I guess that's useless/dumb.  But if there's some kind of auto-base usage level, we could make these optional when whatever the heroku ENV declares to exist.